scripts/build.py

```diff
@@ -10,35 +10,44 @@
 from datetime import datetime
 from string import StringFileConverter
 
-PathVar = os.environ.get('Path')
-Paths = PathVar.split(';')
-PATH = ""
-for candidatePath in Paths:
-    if "devkitARM" in candidatePath:
-        PATH = candidatePath
-        break
-if PATH == "":
-	print('DevKit does not exist in your Path variable.\nChecking default location.')
-	PATH = 'C://devkitPro//devkitARM//bin'
-	if os.path.isdir(PATH) == False:
-		print("...\nDevkit not found.")
-		sys.exit(1)
-	else:
-		print("Devkit found.")
+if sys.platform == 'linux':
+    PREFIX = 'arm-none-eabi-'
+    AS = (PREFIX + 'as')
+    CC = (PREFIX + 'gcc')
+    LD = (PREFIX + 'ld')
+    GR = ("grit")
+    ARP = ('armips')
+    OBJCOPY = (PREFIX + 'objcopy')
+else:
+    PathVar = os.environ.get('Path')
+    Paths = PathVar.split(';')
+    PATH = ""
+    for candidatePath in Paths:
+        if "devkitARM" in candidatePath:
+            PATH = candidatePath
+            break
+    if PATH == "":
+        print('DevKit does not exist in your Path variable.\nChecking default location.')
+        PATH = 'C://devkitPro//devkitARM//bin'
+        if os.path.isdir(PATH) == False:
+            print("...\nDevkit not found.")
+            sys.exit(1)
+        else:
+            print("Devkit found.")
 
-PREFIX = '/arm-none-eabi-'
-AS = (PATH + PREFIX + 'as')
-CC = (PATH + PREFIX + 'gcc')
-LD = (PATH + PREFIX + 'ld')
-GR = ("deps/grit.exe")
-ARP = ('armips')
-OBJCOPY = (PATH + PREFIX + 'objcopy')
+    PREFIX = '/arm-none-eabi-'
+    AS = (PATH + PREFIX + 'as')
+    CC = (PATH + PREFIX + 'gcc')
+    LD = (PATH + PREFIX + 'ld')
+    GR = ("deps/grit.exe")
+    ARP = ('armips')
```

Comment:
I haven't changed this line in windows case. shouldn't it be `'deps/armips.exe'`?